frontend/public/config.production.json

```diff
@@ -1,34 +1,34 @@
 {
   "chains": {
-    "28": {
-      "identifier": 28,
-      "requestManagerAddress": "0x49B600bA97e3D7E5e1f101C2Dd4f49A80A6502E4",
-      "fillManagerAddress": "0xeB5D11D41d68429D82e274f4FFEE94cc1aC950d2",
-      "explorerTransactionUrl": "https://blockexplorer.rinkeby.boba.network/tx/",
-      "rpcUrl": "https://rinkeby.boba.network/",
-      "name": "Boba Rinkeby",
+    "288": {
+      "identifier": 288,
+      "requestManagerAddress": "0x8D7ca47D7d0D9951CEdFC1E6eDA598268dAADE78",
+      "fillManagerAddress": "0x4e32CEc57BCaE80928f58D3dbD84550D397F5416",
+      "explorerTransactionUrl": "https://bobascan.com/tx/",
+      "rpcUrl": "https://lightning-replica.boba.network",
```

Comment:
I think we should use `replica.boba.network` (I remember we had issues with `lightning-replica.boba.network` before and 
`replica.boba.network` was then recommended by Boba people).

Reply:
I checked the scores of the RPC's here https://chainlist.org/ and therefore i decided to go with `lightning-replica.boba.network`.

Reply:
I just saw this on their discord channel:
```
Mainnet RPC
Read RPC: https://lightning-replica.boba.network/
Write RPC: https://mainnet.boba.network/
```
Should we maybe use `https://mainnet.boba.network/` since that is their recommended write RPC endpoint?

Reply:
Switching over to `replica.boba.network` as suggested by the whole team.